RecoLocalMuon/GEMRecHit/test/runGEMRecHitProducer_cfg.py

```diff
@@ -42,6 +42,14 @@
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
```

Comment:
Hi Slava

two things ... 
1) since i was working on the factorization of the pull request I created two files within SimMuon/RPCDigitizer to test within the RPC Package whether everything works fine: 
- .../python/customizeRPCDigi.py (--> stolen from customizeGEMDigi.py and adapted to have only RPC+DT+CSC only digitization. I find it useful to have an easy way to have only muon digitization)
- .../test/runRPCDigiProducer_cfg.py

2) i wrote the fixRPCConditions and fixDTConditions and fixCSCConditions to patch the upgrade global tags such that we can work on the upgrade of the muon system in 75X. Looking at Configurations/AlCa/python/autoCond.py the condition experts have put just placeholders. Once we have good conditions available most of it can be removed (note that for RPC it is not only loading different conditions, but also loading a different Digitization model) ... but it seems to be a pretty though road to get there ... So for now i prefer to have this solution.

But for those two points I will leave it up to you to decide where to put the code or how to deal with this kind of situations. I just committed how I have dealt with it. Maybe for maintenance it is better to have just one customize file and i put the one definition I added in customizeRPCDigi.py inside customizeGEMDigi.py?  
Let me know.
greets
Piet